go/mysql/auth_server_static_test.go

```diff
@@ -168,6 +171,29 @@ func hupTest(t *testing.T, tmpFile *os.File, oldStr, newStr string) {
 	}
 }
 
+func hupTestWithRotation(t *testing.T, tmpFile *os.File, oldStr, newStr string) {
+	*mysqlAuthServerStaticReloadInterval = time.Millisecond * 1
```

Comment:
nit: `1 * time.Millisecond`. Actually, it may be better to make it 10.

Reply:
Done.